iis-wf/iis-wf-transformers/src/test/java/eu/dnetlib/iis/wf/transformer/export/identifier/documenttodataset/DocDatasetToMdStoreWorkflow.java

```diff
@@ -0,0 +1,22 @@
+package eu.dnetlib.iis.wf.transformer.export.identifier.documenttodataset;
+
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import eu.dnetlib.iis.common.AbstractOozieWorkflowTestCase;
+import eu.dnetlib.iis.common.IntegrationTest;
+import eu.dnetlib.iis.common.OozieWorkflowTestConfiguration;
+
+/**
+* @author Łukasz Dumiszewski
+*/
+@Category(IntegrationTest.class)
+public class DocDatasetToMdStoreWorkflow extends AbstractOozieWorkflowTestCase {
```

Comment:
I think that class names for integration tests should end with `Test`. Otherwise it would not be run by `mvn verify`:

```
                    <groupId>org.apache.maven.plugins</groupId>
                    <artifactId>maven-failsafe-plugin</artifactId>
                    <version>2.18.1</version>
                    <executions>
                        <execution>
                            <id>default-integration-test</id>
                            <goals>
                                <goal>integration-test</goal>
                                <goal>verify</goal>
                            </goals>
                            <configuration>
                                <groups>eu.dnetlib.iis.common.IntegrationTest</groups>
                                <includes>
                                    <include>**/*Test.java</include>
                                </includes>
                            </configuration>
                        </execution>
                    </executions>
```